index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <div id="intro">
+        <div id="intro2">
```

Comment:
I shouldn't have used intro 2 as an id  as it doesn't explain what the code actually does . Instead I should have named it something more appealing and something that explains what the div actually does.